packages/block-library/src/table/state.js

```diff
@@ -18,7 +23,7 @@ export function createTable( {
 	return {
 		body: times( rowCount, () => ( {
 			cells: times( columnCount, () => ( {
-				content: [],
+				content: richTextStructure.create(),
```

Comment:
Will it work in the environment which doesn't have DOM (React Native)?

Reply:
Sure, creating an empty structure can be DOM independent.